qml/pages/NewChatPage.qml

```diff
@@ -0,0 +1,394 @@
+                                id: contactListItem
+
+                                opacity: visible ? 1 : 0
+                                Behavior on opacity { FadeAnimation {} }
+
+                                pictureThumbnail {
+                                    photoData: (typeof display.profile_photo !== "undefined") ? display.profile_photo.small : {}
+                                }
+                                width: parent.width
+
+                                primaryText.text: Emoji.emojify(Functions.getUserName(display), primaryText.font.pixelSize, "../js/emoji/")
+                                prologSecondaryText.text: "@" + ( display.username !== "" ? display.username : display.id )
+                                tertiaryText {
+                                    maximumLineCount: 1
+                                    text: Functions.getChatPartnerStatusText(display.status["@type"], display.status.was_online);
+                                }
+
+                                onClicked: {
+                                    contactsListView.newChatInitiated(index);
+                                }
+
+                                Connections {
+                                    target: contactsListView
+
+                                    onNewChatInitiated: {
+                                        if (index === currentIndex) {
+                                            contactListItem.visible = false;
+                                        } else {
+                                            contactListItem.visible = true;
+                                        }
+                                    }
+                                }
+
+                                Connections {
+                                    target: contactsSearchField
+                                    onFocusChanged: {
+                                        if (contactsSearchField.focus) {
+                                            contactListItem.visible = true;
+                                        }
+                                    }
+                                }
+                            }
+
+                            Column {
+                                id: selectChatTypeColumn
+                                visible: !contactListItem.visible
+                                opacity: visible ? 1 : 0
+                                Behavior on opacity { FadeAnimation {} }
+                                width: parent.width
+                                height: contactListItem.height
+
+                                Item {
+                                    width: parent.width
+                                    height: parent.height - chatTypeSeparator.height
+
+                                    Rectangle {
+                                        anchors.fill: parent
+                                        opacity: 0.3
```

Comment:
Opacity `0.3` looks unusual. Pretty much all Sailfish OS UI elements have opacity `0`, `0.2`, `0.4`, `0.6`, `0.8` or `1`.

Reply:
Is that documented somewhere? But OK, will adjust it. :D

Reply:
Don't know about documentation but recent `Theme` defines constants `opacityFaint`, `opacityLow`, `opacityHigh` and `opacityOverlay` which are  `0.2`, `0.4`, `0.6` and `0.8`, respectively. If you don't care about compatibility with earlier versions of Sailfish OS (which you don't) then you can use those constants too.

Reply:
I wasn't aware of these constants, thanks! I changed things accordingly and will change the other opacity values as soon as I touch the respective code areas...